hls4ml/converters/onnx_to_hls.py

```diff
@@ -44,7 +44,11 @@ def get_global_input_shape(graph, inp):
     Raises:
         StopIteration:  If the global input name is not found
     """
-    inp_shape = next(x.type.tensor_type.shape.dim for x in graph.input if x.name == inp)
+    try:
```

Comment:
I don't really like that get_global_input_shape now returns the global output or input shape. Let me see if I can fix it in a different way.